packages/e2e/local-network/templates/babbage/byron-configuration.yaml

```diff
@@ -120,7 +118,7 @@ defaultBackends:
 # be specified in the defaults below or overridden on a per-scribe basis here.
 setupScribes:
   - scKind: FileSK
-    scName: "logs/testnet.log"
+    scName: 'logs/testnet.log'
```

Comment:
This formatting is being inherited from an upstream source, so probably best to ignore anything in `packages/e2e/local-network/templates/`